must-gather/collection-scripts/gather_node_info

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+# Expect base collection path as an argument
+BASE_COLLECTION_PATH=$1
+
+# Use PWD as base path if no argument is passed
+if [ "${BASE_COLLECTION_PATH}" = "" ]; then
+    BASE_COLLECTION_PATH=$(pwd)
+fi
+
+# Command List
+commands_get=()
+
+# lvm commands
+commands_get+=("lvm lvs")
+commands_get+=("lvm lvdisplay")
+commands_get+=("lvm vgs")
+commands_get+=("lvm vgdisplay")
+commands_get+=("lvm pvs")
+commands_get+=("lvm pvdisplay")
+commands_get+=("lvm version")
+commands_get+=("cat /etc/topolvm/lvmd.yaml")
+
+# collection path for lvm commands
+COMMAND_OUTPUT_FILE="${BASE_COLLECTION_PATH}/node_info"
+
+printf "collecting output of node related commands" | tee -a "${BASE_COLLECTION_PATH}/gather-debug.log"
+
+# Create the command, push all the commands to a temp file
+COMMAND_INPUT_FILE="${BASE_COLLECTION_PATH}/temp_file"
+printf "chroot /host" >> "${COMMAND_INPUT_FILE}"
+
+STRING="\\\n"
+for command_get in "${commands_get[@]}"; do
+     # shellcheck disable=SC2129
+     printf "\nprintf \"output of command: %s\"\n" "${command_get}" >> "${COMMAND_INPUT_FILE}"
+     printf "printf \"%s\"\n" "${STRING}" >> "${COMMAND_INPUT_FILE}"
+     printf "%s\n" "${command_get}" >> "${COMMAND_INPUT_FILE}"
+     printf "printf \"%s\"\n" "${STRING}" >> "${COMMAND_INPUT_FILE}"
+done
+
+node=$(oc get nodes --no-headers | awk '/\yReady\y/{print $1}')
```

Comment:
- maybe `oc get nodes -oname` will be good enough?

Reply:
Done